lib/templates/bootstrap/textbox.js

```diff
@@ -22,6 +22,10 @@ function textbox(locals) {
     helpBlockStyle = stylesheet.helpBlock.error;
   }
 
+  if(locals.multiline && textboxStyle.height === 36 ){
```

Comment:
I'm a bit skeptical about this comparison `textboxStyle.height === 36`. Looks like it could be a problem for other custom textbox styles.